doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -50,39 +50,54 @@ While this Reference Architecture is being developed, Gaps will be identified th
 <a name="6.2.3"></a>
 ### 6.2.3 Kubernetes as a VM-based VNF Orchestrator
 
-> In order to support a transition from VNFs only to VNFs and CNFs in the same environment.
+> **Related requirements:** None.
+
+> **Baseline project:** _Kubernetes v1.18.6_, _Kubevirt v0.30.5_
+
+> **Gap description:** Kubernetes and at least on CRI compliant runtime shuld support the runing of VNF-s without completly changing the VNF-s architecture and deployment artifacts.
 
 
 <a name="6.2.4"></a>
 ### 6.2.4 Multiple network interfaces on Pods
 
-> As well as having multiple network interfaces on Pods (e.g. Multus), need to support different network interfaces in different Pods using different CNI plugins within the same cluster.
+> **Related requirements:** [RM Chapter 4.2.2](../../../ref_model/chapters/chapter04.md#422-virtual-network-interface-specifications)
```

Comment:
[RM Chapter 4.2.2](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#422-virtual-network-interface-specifications) doesn't really require multiple network interfaces. What is stated is:
`Note, the number of virtual network interfaces, aka vNICs, associated with a virtual compute instance, is directly related to the number of vNIC extensions declared for the environment.`

I think we need a specific requirement for this in RA2 Ch02.

Reply:
We already have a list of required interface setups in [2.2.2 Virtual Network Interface Specifications](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter02.md#222-virtual-network-interface-specifications) where each requires multiple interface support.

Reply:
@CsatariGergely These are not about multiple interfaces but rather the different speeds that can be selected for an interface.

Reply:
Maybe improve the old text if needed, but not delete it.